fsspec/fuse.py

```diff
@@ -1,32 +1,44 @@
+from __future__ import print_function
```

Comment:
We do not support Python 2, this is redundant

Reply:
Oops, I wrote this last year when python 2 was still supported. Will delete it.